### PR TITLE
Updated  the Readme

### PR DESCRIPTION
**Pull request by wct9667**

### CODE REVIEW

**Comment by BlckHawker** [comment]
Can you add a description on what is different from this version compared to the one in main?

**Comment by wct9667** [comment]
This readme is more in depth within each function: and promotes adding to certain sections as more functions are added. It also contains some of the important code for additional explanation, while also providing more emphasis on the .env and setup of the project.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+		"label": "Blue"
+		},
+		{
+		"id": "1242838265115971634",
+		"label": "Test"
+		}
+	]
+}
+```
+`testServer` Is the id of the server that the bot is running/testing on.
```
Comment (4rt3ry) [inline readme.md:57]: fix funny capitalizations

**Review by 4rt3ry — CHANGES_REQUESTED** [review]
fix some miniscule typos

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+`clientId` Is the id of the bot application.
+`devs` Are the discord Id's for the developers, is useful if certain features are dev only.
+`rolesChannelId` Is the id of the discord channel that the roles button will populate when the bot is started.
+`roles` Is the data for each role, including an `id`  for identification and a `label` for display.
```
Comment (4rt3ry) [inline readme.md:61]: fix funny capitalizations

**Pull request merged**